.github/workflows/build-test-prValidation.yml

```diff
@@ -101,6 +101,53 @@ jobs:
           -Dpekko.log.timestamps=true \
           validatePullRequest
 
+  pekko-classic-remoting-tests:
+    name: Pekko Classic Remoting Tests
+    runs-on: ubuntu-22.04
+    if: github.repository == 'apache/pekko'
+    strategy:
+      fail-fast: false
+      matrix:
+        command:
+          - cluster/test distributed-data/test cluster-tools/test cluster-metrics/test
```

Comment:
need `sbt cluster/MultiJvm/test` too

Reply:
https://github.com/apache/pekko/pull/1592 covers that. It looks like running the multiJvm tests will be more complicated. The aim of this small change is that the test modified in this PR runs in the PR CI - because at the momemt, it does not.